code/__DEFINES/rust_g.dm

```diff
@@ -268,6 +268,10 @@
 /proc/rustg_unix_timestamp()
 	return text2num(RUSTG_CALL(RUST_G, "unix_timestamp")())
 
+// Toast Operations //
+
+#define rustg_create_toast(title, body) RUSTG_CALL(RUST_G, "create_toast")(title, body)
```

Comment:
Dunno what else we would ever use this for, but its a method if we think of uses for it.

Maybe the GC ref finder?

Reply:
I'll implement that when I can later.

Reply:
Done in latest commit